python/pythonmonkey/builtin_modules/XMLHttpRequest.js

```diff
@@ -376,7 +420,6 @@ class XMLHttpRequest extends XMLHttpRequestEventTarget
       return;
     if (this.#timedOutFlag) // step 2
       return this.#reportRequestError('timeout', new DOMException(e.toString(), 'TimeoutError'));
-    console.error(e); // similar to browsers, print out network errors even then the error will be handled by `xhr.onerror`
```

Comment:
Do we need to print out network errors into terminal even the error handled?

Reply:
I don't think so. The browser is unique in that it has a debug console, but in Python, the console is the main user interface.